src/server/resolvers/index.ts

```diff
@@ -1,5 +1,6 @@
 import { UserInputError, AuthenticationError } from 'apollo-server-express';
 import { ObjectID } from 'mongodb';
+import { async } from 'node-ical';
```

Comment:
@BaibhaVatsa what is this doing here? I'm not quite familiar with this, and wouldn't this potentially cause some naming conflicts?

Reply:
Wait lmao. I don't think I added this manually.